src/manual/task-execution/worker-types.md

```diff
@@ -0,0 +1,36 @@
+---
+filename: task-execution/worker-types.md
+title: Worker Types
+order: 30
+---
+
+Worker types can serve a few different purposes within Taskcluster.
+
+Different worker types may have different system configurations. For example,
+one worker type may execute tasks in a Windows environment, while another
+executes tasks within an iOS application context. More subtly, two worker types
+might execute the same "kind" of tasks, but in instances with varying resources
+such as memory and CPU, and correspondingly varied costs.
```

Comment:
I edited the bit up above (which was where I talked about configuring workers of a workerType identically):

>  In practice, this 
does not matter, as all workers with the same `<provisionerId>/<workerType>`
are typically configured identically, or at least so similar that it will not
matter which worker executes a particular task. For example, it is common to
use a variety of similar AWS instance types for a single worker type, allowing
the AWS provisioner to select the most cost-effective option based on current
spot prices.

and in this section, hopefully clearer:

> There are a few different reasons to create new Taskcluster worker types.
> 
> Different worker types may correspond to different system configurations. For 
example, one worker type may execute tasks in a Windows environment, while
another executes tasks within an iOS application context. Thus adding a new platform
would require adding a new worker type to execute tasks on that platform.